zarr/tests/test_storage_v3.py

```diff
@@ -280,7 +280,7 @@ def test_rename_nonexisting(self):
 
     def test_get_partial_values(self):
         store = self.create_store()
-        store.supports_efficient_get_partial_values in [True, False]
+        assert store.supports_efficient_get_partial_values in [True, False]
```

Comment:
That's one of the most useful bugbear rules. It finds such issues much more often than I would have thought.